lib/next_ls.ex

```diff
@@ -592,6 +592,16 @@ defmodule NextLS do
       end
 
     {:reply, reply, lsp}
+  rescue
+    e ->
+      GenLSP.notify(lsp, %GenLSP.Notifications.WindowShowMessage{
+        params: %GenLSP.Structures.ShowMessageParams{
+          type: GenLSP.Enumerations.MessageType.warning(),
+          message: "[Next LS] #{command} command has crashed"
```

Comment:
I would also log the exception, and the show message can be `[Next LS] #{command} has failed, see the logs for more details`